SPECS/ucx/ucx.spec

```diff
@@ -3,47 +3,53 @@
 %bcond_with    cuda
 %bcond_with    gdrcopy
 %bcond_without ib
+#knem disable until MOFED Driver is avialable
+%bcond_with    knem
 %bcond_without rdmacm
 %bcond_with    rocm
 %bcond_with    ugni
+#xpmem disable until MOFED Driver is avialable
 %bcond_with    xpmem
 %bcond_with    vfs
+%bcond_with    mad
+%bcond_with    ze
+%bcond_without mlx5
 
 Summary:        UCX is a communication library implementing high-performance messaging
 Name:           ucx
-Version:        1.15.0
-Release:        1%{?dist}
+Version:        1.18.0
+Release:        3%{?dist}
 License:        BSD
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
-Group:          System Environment/Security
+Group:          System Environment/Libraries
 URL:            http://www.openucx.org
-Source0:        https://github.com/openucx/%{name}/releases/download/v%{version}/ucx-%{version}.tar.gz
-
+Source:         https://github.com/openucx/%{name}/releases/download/v1.18.0-rc1/ucx-1.18.0.tar.gz
```

Comment:
Could you bring back the using `%{version}` for the source URL? This way it gets automatically updated with the version, so we don't accidentally keeping it and `cgmanifest.json` up-to-date.